src/llm/servable.cpp

```diff
@@ -225,6 +227,17 @@ std::string wrapTextInServerSideEventMessage(const std::string& text) {
     ss << "data: " << text << "\n\n";
     return ss.str();
 }
+std::string getBodyString(const ovms::HttpPayload& payload) {
+    if (spdlog::default_logger_raw()->level() == spdlog::level::debug) {
+        auto& json = payload.parsedJson;
+        rapidjson::StringBuffer buffer;
+        rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
+        json->Accept(writer);
+        return buffer.GetString();
+    }
```

Comment:
I don't like the fact that we create a dependency between return value and logging level. 
If this method is only used above in `loadRequest`, then maybe rename it to `logRequestBody` and call `SPD_LOGGER_DEBUG` here to keep it all in the context of logging.